assets/js/theme/common/utils/banner-utils.js

```diff
@@ -0,0 +1,27 @@
+import { isBrowserIE } from './ie-helpers';
+
+const bannerUtils = {
```

Comment:
what's the reason behind creating and importing object? Do expect to add more methods further?

Reply:
Yes, i do. I believe its quite possible that in the future we will need to handle different types of events related to banners.